kedro-org/kedro-viz
# Feature: show matplotlib dataset pngs

## Description

Resolves #783.

Massive thank you to @rashidakanchwala for helping so much with this one.

## Development notes

There are a lot of changes here, and that's because I've renamed several files to adopt a more generic approach to the modal formerly named `PlotlyModal`. That's now simply `MetadataModal`, since we may want to put other things in there.

## QA notes

I've added a `Matplotlib Image` node to the demo project, so you'll be able to see the implementation if you open Gitpod and take a look at the preview link.

## Checklist



<a href="https://gitpod.io/#https://github.com/kedro-org/kedro-viz/pull/887"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
